components/dot_menu/dot_menu.jsx

```diff
@@ -288,6 +291,37 @@ export default class DotMenu extends React.PureComponent {
                 );
             });
 
+        const pluginIntegrationsItems = this.props.pluginIntegrations.filter((item) => {
+            return item.location_type === PluginLocation.PLUGIN_LOCATION_POST_ACTION;
+        }).map((item) => {
+            return (
+                <Menu.ItemAction
+                    text={item.text}
+                    key={item.location_id + '_' + item.text}
+                    onClick={() => {
+                        doPluginAction({
+                            wish: {
+                                url: item.wish.url,
+                            },
```

Comment:
`doPluginAction`'s signature is `export async function doPluginAction(pluginId, requestURL, body)`, yet we are passing one argument as an object here. Does something need to change on the receiving function's side?

Reply:
Yes, this is something I missed. Will fix.